composer.json

```diff
@@ -19,7 +19,8 @@
     "autoload": {
         "psr-4": {
             "PayU\\Alu\\": "src/PayU/Alu/",
-            "PayU\\Alu\\Exceptions\\": "src/PayU/Alu/Exceptions"
```

Comment:
I'm thinking we could drop support for PHP 5.6 for new versions of our library. 
@drealecs What do you think about this?

Also, the readme is stilll saying the minimum version of PHP is 5.3. That would need to be updated in any case.

Reply:
I have changed the PHP version from README. Your observation is good, I've added it on future task.

Reply:
bumping the minimum version to PHP 7.0 in a separate PR sounds good.